packages/trader/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -1,3 +1,5 @@
+@import '../base/index';
+@import '../inline-icons';
```

Comment:
what's the idea behind this? To only import shared css files that are needed in this css file?

Reply:
To import mixins, variables and sass functions, not css files. The `inline-icon` is used in this file as well.